pages/docs/components/typography/text.mdx

```diff
@@ -51,25 +52,6 @@ width of the viewport or `maxWidth` prop.
 </Text>
 ```
 
-Likewise, if you'd like to truncate the text after a specific number of lines,
-pass the `noOfLines` prop and set it to the desired no of lines.
```

Comment:
This "Likewise..." section is a little redundant because we already introduce the `noOfLines` above. 

If we look at the Heading docs, it does not have this section for `noOfLines`

Reply:
Good catch! I'd love to keep the responsive example though. Maybe you can add it to the example above. :)

Reply:
@noobinthisgame Sure, I have updated the example to include the responsive version too. If this looks like a good example, I can also add this to the `Heading` component if we'd like since they both have a similar Truncate Text section.

Reply:
Thanks! I think that it is enough if we have it in one place. We can add it later if needed :)